### ISSUE TITLE
Parse markdown front matter in nunjucks template

### ISSUE THREAD

**Issue by chromadesign** [11ty/eleventy#1243]
**Describe the bug**
I use Forestry.io for clients to update their 11ty content. Fields get written to front matter, including Markdown textarea fields. Unfortunately I can't get those markdown fields rendered with my nunjucks templates. It is always output as is and not parsed with Markdown. Is there any way to make this work besides a custom filter?

**To Reproduce**
Put markdown in front matter nunjucks template.

**Environment:**
- Windows
- Eleventy version: 0.11.0

**Comment by nhoizey** [comment]
@chromadesign do you use the content body in Forestry?

I have this line in my forestry configuration: `hide_body: false`

The body is editable, and goes into the Markdown file body, not in the front matter.

Here's the configuration for [this blog](https://precious-prana.com/blog/):
https://github.com/nhoizey/precious-prana.com/blob/master/.forestry/front_matter/templates/billet.yml

Here is one of the blog posts:
https://github.com/nhoizey/precious-prana.com/blob/master/src/blog/2020-04-30-decouvrez-le-bain-de-gong-pour-apprivoiser-et-apprecier-la-guerison-par-le-son.md

**Comment by chromadesign** [comment]
Yes, partially. The "problem" is that I use a repeatable field group in Forestry which results in nested front matter, so there will be multiple instances of Markdown which is why I can't use the body field for this anyhow.

**Comment by onebitrocket** [comment]
@nhoizey thanks for this. works really well.

I'd lost a few hours to this and trying this solution https://github.com/11ty/eleventy/issues/236 adding `templateEngineOverride: njk,md` to the layout njk file caused the body to be output as escaped html
